haskell/src/Monocle/Backend/Index.hs

```diff
@@ -278,11 +279,50 @@ configDoc = BH.DocId "config"
 -- | Upgrade to config v1 (migrate legacy GH crawler to the new API)
 upgradeConfigV1 :: QueryM ()
```

Comment:
I'll add test case for it in a follow-up commit.

Reply:
Thanks. for the test case I think we should force the config version to 0 by directly calling the indexDocument function.